src/components/ReplyComment/hooks/useReplyComment.js

```diff
@@ -132,7 +135,6 @@ const useReplyComment = ({itemProp, indexFeed, dataFeed, updateParent, page, get
     }
     return [];
   };
-
   const showChildrenCommentView = async (itemReply) => {
     const itemParentProps = await {
       ...itemProp,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `showChildrenCommentView` now uses an async-await pattern. This could potentially lead to unhandled promise rejections if there are any errors in the awaited operations. Consider adding a try-catch block to handle potential errors.

```diff
+  const showChildrenCommentView = async (itemReply) => {
+    const itemParentProps = await {
+      ...itemProp,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->